chainercv/datasets/cub/cub_keypoint_dataset.py

```diff
@@ -93,8 +94,9 @@ def get_example(self, i):
 
         if self.crop_bbox:
             bbox = self.bboxes[i]  # (x, y, width, height)
-            img = img[bbox[1]: bbox[1] + bbox[3], bbox[0]: bbox[0] + bbox[2]]
-            keypoint[:, :2] = keypoint[:, :2] - np.array([bbox[0], bbox[1]])
+            img =\
+                img[:, bbox[1]: bbox[1] + bbox[3], bbox[0]: bbox[0] + bbox[2]]
```

Comment:
It is better to make another PR for this bug fix. It should be sent for `master` branch.